.github/workflows/nocgo.yml

```diff
@@ -0,0 +1,37 @@
+name: Go
```

Comment:
I would just keep only one file `go.yml` as we don't build any binary and there is no difference `cgo` or `nocgo`